src/main/webapp/js/communication/geppettoJupyter/PythonControlledCapability.js

```diff
@@ -86,6 +86,7 @@ define(function (require) {
                     this.handleChange = (this.props.handleChange == undefined) ? this.handleChange.bind(this) : this.props.handleChange.bind(this);
                     this.handleUpdateInput = this.handleUpdateInput.bind(this);
                     this.handleUpdateCheckbox = this.handleUpdateCheckbox.bind(this);
+                    this.setErrorAlert = this.setErrorAlert.bind(this);
```

Comment:
Is this actually needed?

Reply:
I promise to read when exactly to bind to this

Reply:
Are you planning to delete it?